examples/algorithms/plot_trust_rank.py

```diff
@@ -0,0 +1,62 @@
+"""
+===================
+TrustRank Algorithm
+===================
+
+TrustRank is an algorithm designed to compute trust scores for nodes in a graph.
+It is particularly useful in applications like web spam detection, where certain nodes (web pages) are
+known to be trustworthy, and the goal is to propagate trust to other nodes based on the graph structure.
+
+How TrustRank Relies on PageRank:
+---------------------------------
+TrustRank uses the personalization feature of PageRank to bias the ranking towards a set of "seed nodes."
+These seed nodes are considered trustworthy and are assigned higher initial scores in the personalization
+vector. The algorithm then propagates trust through the graph using the same iterative process as PageRank.
+
+Key Steps:
+
+1. Define a set of seed nodes that are considered trustworthy.
+2. Create a personalization vector where seed nodes are assigned equal trust,
+   and non-seed nodes are assigned zero.
+3. Run the PageRank algorithm with the personalization vector to compute trust scores for all nodes.
+
+References
+----------
+.. [1] Gyongyi, Zoltan; Garcia-Molina, Hector (2004).
+    "Combating Web Spam with TrustRank"
+    http://ilpubs.stanford.edu:8090/770/1/2004-52.pdf
+.. [2] Krishnan, Vijay; Raj, Rashmi
+    "Web Spam Detection with Anti-Trust Rank"
+    http://i.stanford.edu/~kvijay/krishnan-raj-airweb06.pdf
```

Comment:
These seem to deal with applications of trustrank rather than the algorithm itself. Personally I'd prefer to delete the references section and instead turn the first instance of TrustRank into a link to the wiki article (see above suggestions for how to do so).